### PR TITLE
fix typing issues

### PR DESCRIPTION
**Pull request by jdkandersson**
The tests seem to be failing, I'm not that familiar with the code, maybe @arturo-seijas could you check this and see what should be changed? Feel free to take over this branch and PR

### CODE REVIEW

**Comment by jdkandersson** [comment]
Proposing @yanksyoon to work on this

File: src/charm.py [hunk 1]
```diff
@@ -171,25 +177,29 @@ def _get_external_port(self) -> int:
...
     def _are_relations_ready(self, _) -> bool:
         """Check if the needed relations are established.
 
         Returns:
             If the needed relations have been established.
         """
-        if not any(
-            rel.app.name.startswith("redis-broker") for rel in self.model.relations["redis"]
+
+        if not not any(
```
Comment (gregory-schiano) [inline src/charm.py:189]: `not not` ?

Comment (arturo-seijas) [reply src/charm.py]: Fixed

**Review by merkata — DISMISSED** [review]
Looks great, nice addition of mypy!

**Review by nrobinaubertin — APPROVED** [review]
LGTM

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Review by merkata — APPROVED** [review]
Great!

**Pull request merged**